### PR TITLE
[Yolov8 detector] Fix example in docstring

### PR DESCRIPTION
**Pull request by cosmo3769**
# What does this PR do?

* Fixed typo: 
-- `yolo_v8_m_coco` to `yolo_v8_m_backbone_coco`
-- `fpn_depth=2.` to `fpn_depth=2`
* Fixed `model.fit`
-- Converted `labels` to `labels_tensor`.
-- `model.fit(images, labels)` to `model.fit(images, labels_tensor)`

## Who can review?

@ianstenbit @jbischof

### CODE REVIEW

**Review by ianstenbit — CHANGES_REQUESTED** [review]
Thanks for the fix! Looks good -- just one little tweak

File: keras_cv/models/object_detection/yolo_v8/yolo_v8_detector.py [hunk 1]
```diff
@@ -362,13 +362,24 @@ class YOLOV8Detector(Task):
         ],
         "classes": [[1, 1, 1]],
     }
+
+    # Convert labels to tensors
```
[the remark this answers is not in the document]
Comment (cosmo3769) [reply keras_cv/models/object_detection/yolo_v8/yolo_v8_detector.py]: Yeah true, fixed it.

**Review by ianstenbit — APPROVED** [review]
Thank you!
